docs/financial/financialentities.md

```diff
@@ -61,3 +61,25 @@ Negative payments (or refunds) will have the bank or processor account on the **
 
 Financial items exist to track how much has been paid against the various line items within an order / contribution.
 In an accounting system this might be called credit-matching.
+
+Each line item will have one or more financial items denoting payments made against it. On creation of the order there
+will be a financial item for each line item and an additional item for every fee amount. Where the preferred flow is used
+this item will have a status of 'Unpaid'. (Note that for historical reasons this may not always be true but it's the goal - [see](https://github.com/civicrm/civicrm-dev-docs/issues/712))
```

Comment:
```suggestion
this item will have a status of 'Unpaid'.

!!! note
    For historical reasons this may not always be true but [it's the goal](https://github.com/civicrm/civicrm-dev-docs/issues/712).
```

Reply:
This would look nicer with the note in an admonition box.